### PR TITLE
ZERH MF Ver2 Implementation

### PR DESCRIPTION
**Pull request by bpark1327**
## Pull Request Description

ZERH MF Ver2 Implementation (https://www.energy.gov/sites/default/files/2023-03/DOE%20ZERH%20Multifamily%20V2%20-%20ERI%20Target%20Procedure%20DRAFT%20March%202023.pdf)

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [x] ~OS-HPXML git subtree has been pulled~
- [x] 301/ES rulesets and unit tests have been updated
- [x] 301validator.xml has been updated (reference EPvalidator.xml)
- [x] Workflow tests have been updated
- [x] Documentation has been updated
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected regression test changes on CI

### CODE REVIEW

**Comment by bpark1327** [comment]
Currently, we assume all water heaters are residential because we require EF/UEF. We may need to accommodate commercial water heaters and allow for thermal efficiency (Et) input.

**Comment by bpark1327** [comment]
@shorowit ZERH MF Ver 2 has the following specification, but we don't currently accommodate the central boiler, so I don't know how to handle it.
image

**Comment by shorowit** [comment]
Need to update mech vent SRE -> ASRE

**Comment by bpark1327** [comment]
> Need to update mech vent SRE -> ASRE

Addressed in commit c150a8b

**Comment by bpark1327** [comment]
Updated to ZERH MF Ver2 released on 11/27/2023

File: rulesets/data/zerh_mf_2_0_lookup.tsv [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+hvac_ac_seer		18	18	18	18	18	18	16	16	16	16	16	14	14	14	14	14	14	14	14	14
+hvac_ashp_seer		18	18	18	18	18	18	16	16	16	16	16	16	16	16	16	16	16	16	16	16
+hvac_gshp_eer		16	16	16	16	16	16	14	14	14	14	14	14	14	14	14	14	14	14	14	14
+hvac_wlhp_eer		15	15	15	15	15	15	15	15	15	15	15	15	15	15	15	15	15	15	15	15
+hvac_chiller_kw_per_ton		0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75	0.75
+hvac_charge_defect_ratio		-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25	-0.25
+hvac_airflow_defect_ratio		-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075	-0.075
+hvac_fan_watts_per_cfm		0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45	0.45
+shared_motor_efficiency		0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85	0.85
+hot_water_distribution_pipe_r_value		3	3	3	3	3	3	3	3	3	3	3	3	3	3	3	3	3	3	3	3
```
[the remark this answers is not in the document]
Comment (shorowit) [reply rulesets/data/zerh_mf_2_0_lookup.tsv:77]: Guidance from DOE: R-3 if there's a shared water heater, R-0 if not. I'll address this.

**Review by shorowit — APPROVED** [review]
Nice work, @bpark1327. Looks good to me.

**Pull request merged**